app/views/shared/_forms-header.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="row bg-light">
+  <div class="col-10">
+    <div class="dropdown">
+      <button class="btn btn-primary dropdown-toggle" type="button" id="dropdownMenuButton1" data-bs-toggle="dropdown" aria-expanded="false">
+        Submitted Forms
+      </button>
+      <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton1">
+        <li><%= link_to "Working", employee_submissions_submissions_path(_type: "working"), class: "dropdown-item" %></li>
+        <li><%= link_to "Medical", employee_submissions_submissions_path(_type: "medical"), class: "dropdown-item" %></li>
+        <li><%= link_to "Expense", employee_submissions_submissions_path(_type: "expense"), class: "dropdown-item" %></li>
+        <li><%= link_to "General", employee_submissions_submissions_path(_type: "general"), class: "dropdown-item" %></li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="col-2">
+    <%= link_to "New From", new_submission_path(_type: "working"), class: "btn btn-primary mt-1 float-end" %>
```

Comment:
do user always submit a working type?

Reply:
at start by default we are passing working form